convert_hf_to_gguf.py

```diff
@@ -281,8 +281,10 @@ def modify_tensors(self, data_torch: Tensor, name: str, bid: int | None) -> Iter
         return [(self.map_tensor_name(name), data_torch)]
 
     def tensor_force_quant(self, name: str, new_name: str, bid: int | None, n_dims: int) -> gguf.GGMLQuantizationType | bool:
+        # 删除未使用的参数，避免警告或错误
         del name, new_name, bid, n_dims  # unused
 
+        # 返回False，表示不进行量化
```

Comment:
Maybe these two comments could be deleted as well? (I checked translation to English, the `#  unused` seems like drives the purpose for the upper one).

Doesn't seem to be related to GLM4 arch.